db/migrate/20180518072400_create_sensor_rules.rb

```diff
@@ -0,0 +1,10 @@
+class CreateSensorRules < ActiveRecord::Migration[5.2]
+  def change
+    create_table :sensor_rules do |t|
+      t.belongs_to :sensor, foreign_key: true
```

Comment:
should also add an index here

Reply:
belongs_to in migration already also included index

Reply:
oh good!